<def f='llvm/llvm/include/llvm/DebugInfo/PDB/GenericError.h' l='17' ll='24'/>
<use f='llvm/llvm/include/llvm/DebugInfo/PDB/GenericError.h' l='30'/>
<use f='llvm/llvm/include/llvm/DebugInfo/PDB/GenericError.h' l='37' c='_ZN4llvm3pdb15make_error_codeENS0_14pdb_error_codeE'/>
<use f='llvm/llvm/lib/DebugInfo/PDB/GenericError.cpp' l='24' c='_ZNK12_GLOBAL__N_116PDBErrorCategory7messageB5cxx11Ei'/>
